packages/react/src/components/Pagination/next/Pagination.stories.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+
+import {
+  withKnobs,
+  array,
+  boolean,
+  number,
+  select,
+  text,
+} from '@storybook/addon-knobs';
```

Comment:
Do we want to start removing knobs if the story is going to be in `carbon-react-next`?

Reply:
Good point, maybe this can be part of our refactor during our mob days next sprint?